packages/firebase_core/firebase_core_web/lib/src/firebase_core_web.dart

```diff
@@ -10,15 +10,20 @@ class FirebaseWebService {
   /// The name which matches the Firebase JS Web SDK postfix.
   String name;
 
+  /// Overrides the created window object.
+  String? override;
```

Comment:
It's not obvious for me how to use this override from the comment. Can we add some details?